ui/pages/send/gas-display/gas-display.js

```diff
@@ -359,12 +360,9 @@ export default function GasDisplay({ gasError }) {
                 ) : (
                   <Typography variant={TypographyVariant.H7} align="left">
                     {t('insufficientCurrencyBuyOrReceive', [
-                      draftTransaction.asset.details?.symbol ?? nativeCurrency,
+                      nativeCurrency,
                       currentNetworkName,
-                      `${t('buyAsset', [
-                        draftTransaction.asset.details?.symbol ??
```

Comment:
In token transfer transactions we should still show the native currency always here